### PR TITLE
feat(readme): update readme description

### PR DESCRIPTION
**Pull request by ferrorenan**
Updates readme organization for better visualization and description isolation

### CODE REVIEW

**Comment by ferrorenan** [comment]
@tolik518 

Hey, how are you?!

I made this PR to leave the readme with better organization of descriptions and better visualization for everyone! What do you think?!

If you approve, could you tag it with the Hacktoberfest tag, please?!

I hope to do more PR for your projects!

File: README.MD [hunk 1]
```diff
@@ -15,21 +15,27 @@ This is the source code behind [blog.returnnull.de](https://blog.returnnull.de).
 [![PHPStorm](code/public/img/stack/phpstorm.png)](https://www.jetbrains.com/de-de/phpstorm/)
 [![Git](code/public/img/stack/git.png)](https://git-scm.com/)
 
+## How to Start Development
+- Clone repository or Fork
```
Comment (tolik518) [inline README.MD:19]: - Fork and Clone the repository

File: README.MD [hunk 2]
```diff
@@ -15,21 +15,27 @@ This is the source code behind [blog.returnnull.de](https://blog.returnnull.de).
...
+- Build images
+  - Build with: `make build`
+  - Building with Docker: `docker compose build`
```
Comment (tolik518) [inline README.MD:22]: Both commands use docker, so the wording should be:

- Build images
  - Build with: `make build` or with `docker compose build`

File: README.MD [hunk 3]
```diff
@@ -15,21 +15,27 @@ This is the source code behind [blog.returnnull.de](https://blog.returnnull.de).
...
+- Run the application: Use `make run` or `docker-compose up -d` to start the containers
+  - To install the dependencies and to update the autoloader (do this everyime you add a new class) `make install` or `docker-compose -f docker/compose/docker-compose-cli.yml run --rm --no-deps php_cli php -d memory_limit=-1 /usr/local/bin/composer install`
+- To stop:
+  - Option 1: `make stop`
+  - Option 2: `docker-compose down --remove-orphans` to stop the containers
```
Comment (tolik518) [inline README.MD:27]: keep the readme standardized, don't use Option 1/Option 2 - write it like above

**Review by tolik518 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**